kubernetes/ansible/roles/sunbird-monitoring/templates/dp_prometheus-adapter.yaml

```diff
@@ -0,0 +1,165 @@
+# Default values for k8s-prometheus-adapter..
+affinity: {}
+
+image:
+  repository: directxman12/k8s-prometheus-adapter-amd64
+  tag: v0.7.0
+  pullPolicy: IfNotPresent
+
+logLevel: 4
+
+metricsRelistInterval: 1m
+
+listenPort: 6443
+
+nodeSelector: {}
+
+priorityClassName: ""
+
+# Url to access prometheus
+prometheus:
+  url: http://'{{core_prom_ip.split(":")[0]}}
+  port: 9090
+  path: ""
+
+replicas: 1
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+resources: {}
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+rules:
+  default: true
+  custom:{}
+# - seriesQuery: '{__name__=~"^some_metric_count$"}'
+#   resources:
+#     template: <<.Resource>>
+#   name:
+#     matches: ""
+#     as: "my_custom_metric"
+#   metricsQuery: sum(<<.Series>>{<<.LabelMatchers>>}) by (<<.GroupBy>>)
+  # Mounts a configMap with pre-generated rules for use. Overrides the
+  # default, custom, external and resource entries
+  existing:
+  external:
+  - seriesQuery: 'kafka_consumergroup_lag_sum'
```

Comment:
Do you want to add query for  all the jobs ?

Reply:
Yes - added query for all the main pipeline jobs